### DISCUSSION TITLE
Any idea why this is happening?

### DISCUSSION THREAD

**Discussion by MasterTro11** [Acly/krita-ai-diffusion#676] [General]
![DSC03435](https://github.com/Acly/krita-ai-diffusion/assets/97849400/0c1878ee-48be-4693-8201-75e0c0f3f838)

![Screenshot 2024-05-01 151836](https://github.com/Acly/krita-ai-diffusion/assets/97849400/afcf3d2b-d967-4919-b28c-93ba9cfb3a2b)

I appreciate your dedication to this app—it's greatly appreciated! I frequently use the object removal feature, but I've noticed some discoloration on the wall. Could you advise if there's a mistake in my approach? I'm specifically using the object removal tool.

**Comment by rbfbatx** [comment]
Try a different VAE. This happens a lot with SDXL.

Doing another refine pas at 45 also helps sometimes; but also sometimes makes it worse so do a several a batch until you get one that is right.

**Reply by MasterTro11** [reply]
Thank you for responding. I'll need to delve deeper into how to adjust those settings. So far, I haven't noticed any variables that can be modified in Krita through the control panel at the bottom right, but I'll continue to explore. Thank you.

**Comment by Acly** [comment]
Color/brightness shifts can happen in these kinds of scenarios. For a possible explanation see [my comment](https://github.com/Acly/krita-ai-diffusion/issues/662#issuecomment-2092485463)

Personally I think it's fastest to manually adjust hue/saturation/value of the generated layer, and possibly smoothen the transition a bit with a soft eraser. I hope this will improve with future models.

**Comment by freecoderwaifu** [comment]
One of my current holy grails of AI is AI color matching and better color consistency in general. There are already a bunch of "traditional" methods for color matching, including Krita's own color transfer and also with G'Mic, or using Clip Studio Paint's color matching and many other ways to do this, but it's still not perfect. 

An AI solution that 1:1 matches colors and other values like brightness and contrast (though arguably those are also kinda sorta not really but kinda also pixel color values) would be great if someone can figure it out, especially for some types of animation. There's a color ControlNet but it also isn't quite there.
